src/mixins/portal/portal.ts

```diff
@@ -231,9 +231,12 @@ export class Portal extends ModulVue implements PortalMixin {
         }
     }
 
-    private toggle(event): void {
-        this.$emit('click', event);
-        this.propOpen = !this.propOpen;
+    private toggle(event: MouseEvent): void {
+        if (this.openTrigger !== MOpenTrigger.Click ||
```

Comment:
Qu'est-ce qu'on veut valider avec cette condition? Qu'on a cliqué que le bouton LEFT? Le toggle est appelé sur un listener attaché au mousedown (ligne 226). Le listener est attaché avec la condition suivante: `(this.openTrigger === MOpenTrigger.Click)`. Je ne vois pas dans quel cas on a besoin de valider `this.openTrigger !== MOpenTrigger.Click `

Reply:
Oublie ça, ça provient de ton merge avec develop